src/test/kotlin/com/whatever/domain/content/service/ContentServiceUnitTest.kt

```diff
@@ -0,0 +1,463 @@
+        every { contentRepository.findContentByIdAndType(memoId, ContentType.MEMO) } returns memo
+        every { coupleRepository.findByIdWithMembers(coupleId) } returns null
+
+        // when
+        val exception = assertThrows<ContentAccessDeniedException> {
+            contentService.getMemo(memoId)
+        }
+
+        // then
+        assertThat(exception.errorCode).isEqualTo(COUPLE_NOT_MATCHED)
+    }
+
+    @Test
+    @DisplayName("메모 작성자가 커플 멤버가 아니면 ContentAccessDeniedException이 발생한다")
+    fun getMemo_memoAuthorNotInCouple() {
+        // given
+        val memoId = 1L
+        val coupleId = 1L
+        val userId = 1L
+        val partnerId = 2L
+        val otherUserId = 3L
+
+        val user = createTestUser(id = userId)
+        val partner = createTestUser(id = partnerId, nickname = "파트너", gender = UserGender.FEMALE, birthYear = 1991)
+        val otherUser =
+            createTestUser(id = otherUserId, nickname = "다른유저", userStatus = UserStatus.SINGLE, birthYear = 1992)
+        val couple = createTestCouple(id = coupleId, user1 = user, user2 = partner)
+        val memo = createTestContent(
+            id = memoId,
+            user = otherUser,
+            title = "다른 사용자의 메모",
+            description = "다른 사용자가 작성한 메모"
+        )
+
+        every { SecurityUtil.getCurrentUserCoupleId() } returns coupleId
+        every { contentRepository.findContentByIdAndType(memoId, ContentType.MEMO) } returns memo
+        every { coupleRepository.findByIdWithMembers(coupleId) } returns couple
+
+        // when
+        val exception = assertThrows<ContentAccessDeniedException> {
+            contentService.getMemo(memoId)
+        }
+
+        // then
+        assertThat(exception.errorCode).isEqualTo(COUPLE_NOT_MATCHED)
+    }
+
+    @Test
+    @DisplayName("명시적으로 커플 ID를 전달하여 메모를 조회할 수 있다")
+    fun getMemo_withExplicitCoupleId() {
+        // given
+        val memoId = 1L
+        val coupleId = 1L
+        val userId = 1L
+        val partnerId = 2L
+
+        val user = createTestUser(id = userId)
+        val partner = createTestUser(id = partnerId, nickname = "파트너", gender = UserGender.FEMALE, birthYear = 1991)
+        val couple = createTestCouple(id = coupleId, user1 = user, user2 = partner)
+        val memo = createTestContent(id = memoId, user = user, isCompleted = true)
+
+        every { contentRepository.findContentByIdAndType(memoId, ContentType.MEMO) } returns memo
+        every { coupleRepository.findByIdWithMembers(coupleId) } returns couple
+        every { tagContentMappingRepository.findAllWithTagByContentId(memoId) } returns emptyList()
+
+        // when
+        val result = contentService.getMemo(memoId, coupleId)
+
+        // then
+        assertThat(result.id).isEqualTo(memoId)
+        assertThat(result.title).isEqualTo("테스트 메모")
+        assertThat(result.description).isEqualTo("테스트 메모 내용")
+        assertThat(result.isCompleted).isTrue()
+        assertThat(result.tagList).isEmpty()
```

Comment:
assert 문 이후에 verify 구문을 통해 mockking 했던 함수들이 잘 불렸는지, 값이 잘 전달됐는지도 체크해보면 좋을 것 같아요
```suggestion
        assertThat(result.tagList).isEmpty()
        
        verify(exactly = 1) {
            contentRepository.findContentByIdAndType(eq(memoId), eq(ContentType.MEMO))
        }
```

추가로 verify를 쓰면서 생각이 든건데, 보통 every 문에서는 any() 로 값을 퉁쳤던 것 같아요 
그리고, verify문에서 해당 로직이 수행되면서 원하는 값이 들어갔는지 eq() 로 검증하구요